html/browsers/browsing-the-web/navigating-across-documents/007.html

```diff
@@ -1,16 +1,29 @@
 <!doctype html>
-<title>Link with onclick javascript url and href navigation </title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigating-across-documents">
+<title>Link with onclick document.write is aborted by href navigation</title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <div id="log"></div>
 <iframe id="test" name="test"></iframe>
-<a target="test" onclick="document.getElementById('test').contentWindow.location = 'javascript:\'abc<script>parent.postMessage(&quot;click&quot;, &quot;*&quot;)</script>\'';" href="href.html">Test</a>
+<a target="test" onclick="document.getElementById('test').contentWindow.document.write('<script>self.postMessage(&quot;click&quot;, &quot;*&quot;)</script>');" href="href.html">Test</a>
```

Comment:
Hmm.  So this totally changes what this test is testing; it used to test javascript: navigation, but now it tests document.open/write.  Why the change?  I'd like to understand that before reviewing the details here.

Reply:
This is a mistake, I need to revisit 007, 008 and 009.